README.md

```diff
@@ -0,0 +1,173 @@
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖 [To Do List] <a name="To Do List: Project setup with WebPack"></a>
+
+**[To Do List]** is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete. You can delete,edit and add more tasks.
+
+## 🛠 Built With <a name="Visual Studio Code"></a>
+
+### Tech Stack <a name="JavaScript HTML & CSS"></a>
+
+- Vite
+- React
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Adding task**
+- **Removing task**
+- **Completing task using check**
+- **Editing Task**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Not Available yet]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+If you are a new developer then you can learn basic UI feature from this project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Browser setup in your computer
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  https://github.com/ngeelani48/react-todo-app.git
+```
+
+### Install
+
+Install this project with:
+
+  -  [Git](https://git-scm.com/downloads)
+  -  [Node](https://nodejs.org/en/download/)
+### Usage
+
+To run the project, execute the following command:
+- After Cloning this repo to your local machine
+- Run  `npm run dev ` in terminal
+- Open live server  Link in browser
+### Run tests
+
+To track linter errors locally follow these steps:  
+
+Download all the dependencies run:
+```
+npm install
+```
+Track CSS linter errors run:
+```
+npx stylelint "**/*.{css,scss}"
+```
+Track JavaScript linter errors run:
+```
+npx eslint "**/*.{js,jsx}"
+```
+Track Jest Test run:
+```
+npm test
+```
+
+### Deployment
+
+You can deploy this project using:
+
+(https://ngeelani48.github.io/To-Do_List/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Muhammad Nafees**
+
+- GitHub: [@ngeelani48](https://github.com/ngeelani48)
+- Twitter: [@twitterhandle](https://twitter.com/ngeelani48)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/m-n-geelani-3446097275/)
+
+👤 **Sane Myburg**
+
+- GitHub: [@SaneMyburg](https://github.com/SaneMyburg)
+- Twitter: [@SAneMyburg](https://twitter.com/SaneMyburg)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/SaneMyburg/)
+
```

Comment:
- [ ]  Kindly update your README file

Reply:
@Tafloninno Thanks for having a deep look, I will make these changes.